php/index.php

```diff
@@ -51,8 +51,13 @@
 });
 
 $app->put('/posts/:id', function($id) use ($app, $crate) {
+	$qry = $crate->prepare("UPDATE guestbook.posts SET text=? WHERE id=?");
+	$qry->bindParam(1, $app->request->post('text'));
+	$qry->bindParam(2, $id);
+	$state = $qry->execute();
+
 	header(CONTENT_TYPE_JSON);
-	echo json_encode(array());
+	echo json_encode(array('success' => $state));
```

Comment:
i think we have to sync the response anyway with the python stuff.
therefore i'll wait till this is finished and go through all the responses then.